docs/libcurl/opts/CURLOPT_HTTPHEADER.3

```diff
@@ -98,6 +98,9 @@ implies that if you tell libcurl to follow redirects
 in the subsequent request. Redirects can of course go to other hosts and thus
 those servers will get all the contents of your custom headers too.
 
+This option must also be used to set the top-level headers of a MIME mail when
+uploading the latter via SMTP or IMAP.
+
```

Comment:
how is this a security concern

Reply:
> how is this a security concern

Thanks for pointing this out: I placed the paragraph in a wrong section.